image/templates/helm/stackrox-secured-cluster/templates/sensor-scc.yaml

```diff
@@ -12,9 +13,11 @@ metadata:
   annotations:
     {{- include "srox.annotations" (list . "securitycontextconstraints" "stackrox-sensor") | nindent 4 }}
     kubernetes.io/description: stackrox-sensor is the security constraint for the sensor
+{{- if eq ._rox.env.openshift 3 }}
```

Comment:
My 2c: leave this as unconditional. This change will only cause pain when having to write upgrade instructions, and unless I am not mistaken, does not provide any real benefit (even if we are adding RBAC bindings, these entries here don't hurt anything we want to accomplish). Also, I thought that we wanted to migrate away from custom SCCs soon-ish. Why would we make any changes to them?

Reply:
Ok, I can remove it. I wanted to avoid that OCP v4 users would get the permission to use the custom SCC from 2 different sources, but you are right, we do not hurt any of our goals making this unconditional.